docs/kyma/04-04-cluster-installation.md

```diff
@@ -156,25 +156,39 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
   Gardener
   </summary>
 
-1. In the left navigation of the Gardener UI, go to the **Secrets** tab and add Secrets to enable provisioning clusters on different architectures. To learn about the requirements for each environment, click the question mark buttons.
+1. Use the Gardener dashboard to configure provider settings.
 
-2. Provision a cluster form the **Clusters** tab. Click the plus sign in the lower-right corner and choose the infrastructure in which you want to provision your cluster. Apply these settings in the following tabs:
+    >**NOTE:** You need to perform these steps only once.
+   
+    * For GCP:
+      * Create a project in Gardener.
+      * Add a [new service account and roles](https://gardener.cloud/050-tutorials/content/howto/gardener_gcp/#create-a-new-serviceaccount-and-assign-roles).
+      >**NOTE**: Run `gcloud auth application-default login` and log in with your Google Cloud credentials to be authenticated by Google SDK.
```

Comment:
That note is not relevant here as we will not use google specific tooling

Reply:
Removed :)